src/code/audio/audio_data.c

```diff
@@ -0,0 +1,895 @@
+    /* 0x4E */ 9.513658f,   // NOTE_EFLAT7
+    /* 0x4F */ 10.07937f,   // NOTE_E7
+    /* 0x50 */ 10.6787205f, // NOTE_F7
+    /* 0x51 */ 11.31371f,   // NOTE_GFLAT7
+    /* 0x52 */ 11.986459f,  // NOTE_G7
+    /* 0x53 */ 12.699211f,  // NOTE_AFLAT7
+    /* 0x54 */ 13.454346f,  // NOTE_A7
+    /* 0x55 */ 14.254383f,  // NOTE_BFLAT7
+    /* 0x56 */ 15.101993f,  // NOTE_B7
+    /* 0x57 */ 16.0f,       // NOTE_C8
+    /* 0x58 */ 16.95141f,   // NOTE_DFLAT8
+    /* 0x59 */ 17.959395f,  // NOTE_D8
+    /* 0x5A */ 19.027315f,  // NOTE_EFLAT8
+    /* 0x5B */ 20.15874f,   // NOTE_E8
+    /* 0x5C */ 21.35744f,   // NOTE_F8
+    /* 0x5D */ 22.62742f,   // NOTE_GFLAT8
+    /* 0x5E */ 23.972918f,  // NOTE_G8
+    /* 0x5F */ 25.398422f,  // NOTE_AFLAT8
+    /* 0x60 */ 26.908691f,  // NOTE_A8
+    /* 0x61 */ 28.508766f,  // NOTE_BFLAT8
+    /* 0x62 */ 30.203985f,  // NOTE_B8
+    /* 0x63 */ 32.0f,       // NOTE_C9
+    /* 0x64 */ 33.90282f,   // NOTE_DFLAT9
+    /* 0x65 */ 35.91879f,   // NOTE_D9
+    /* 0x66 */ 38.05463f,   // NOTE_EFLAT9
+    /* 0x67 */ 40.31748f,   // NOTE_E9
+    /* 0x68 */ 42.71488f,   // NOTE_F9
+    /* 0x69 */ 45.25484f,   // NOTE_GFLAT9
+    /* 0x6A */ 47.945835f,  // NOTE_G9
+    /* 0x6B */ 50.796845f,  // NOTE_AFLAT9
+    /* 0x6C */ 53.817383f,  // NOTE_A9
+    /* 0x6D */ 57.017532f,  // NOTE_BFLAT9
+    /* 0x6E */ 60.40797f,   // NOTE_B9
+    /* 0x6F */ 64.0f,       // NOTE_C10
+    /* 0x70 */ 67.80564f,   // NOTE_DFLAT10
+    /* 0x71 */ 71.83758f,   // NOTE_D10
+    /* 0x72 */ 76.10926f,   // NOTE_EFLAT10
+    /* 0x73 */ 80.63496f,   // NOTE_E10
+    /* 0x74 */ 85.42976f,   // NOTE_F10
+    /* 0x75 */ 0.055681f,   // NOTE_BFLATNEG1
+    /* 0x76 */ 0.058992f,   // NOTE_BNEG1
+    /* 0x77 */ 0.0625f,     // NOTE_C0
+    /* 0x78 */ 0.066216f,   // NOTE_DFLAT0
+    /* 0x79 */ 0.070154f,   // NOTE_D0
+    /* 0x7A */ 0.074325f,   // NOTE_EFLAT0
+    /* 0x7B */ 0.078745f,   // NOTE_E0
+    /* 0x7C */ 0.083427f,   // NOTE_F0
+    /* 0x7D */ 0.088388f,   // NOTE_GFLAT0
+    /* 0x7E */ 0.093644f,   // NOTE_G0
+    /* 0x7F */ 0.099213f,   // NOTE_AFLAT0
+};
+
+u8 gDefaultShortNoteVelocityTable[] = {
+    12, 25, 38, 51, 57, 64, 71, 76, 83, 89, 96, 102, 109, 115, 121, 127,
+};
+
+u8 gDefaultShortNoteGateTimeTable[] = {
+    229, 203, 177, 151, 139, 126, 113, 100, 87, 74, 61, 48, 36, 23, 10, 0,
+};
+
+AdsrEnvelope gDefaultEnvelope[] = {
+    { 1, 32000 },
+    { 1000, 32000 },
+    { -1, 0 },
+    { 0, 0 },
+};
+
+NoteSubEu gZeroNoteSub = { 0 };
+
+NoteSubEu gDefaultNoteSub = {
+    { 1, 1, 0, 0, 0, 0, 0, 0 }, { 0 }, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+};
+
+u16 gHeadsetPanQuantization[64] = {
+    60, 58, 56, 54, 52, 50, 48, 46, 44, 42, 40, 38, 36, 34, 32, 30, 28, 26, 24, 22, 20, 18,
+    16, 14, 12, 10, 8,  6,  4,  2,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
+    0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
+};
+
+s32 D_801D58A4 = 0;
+
+// clang-format off
+s16 D_801D58A8[] = {
```

Comment:
Any idea what this data is?

Reply:
This has something to do with the parameters used in adpcm (Adaptive differential pulse-code modulation) which is used to encode audio data.

This block of data is very confusing however, because the values and how it's used doesn't make sense given the context. Interestingly, this exact piece of data is used in the same way for both SM64 and OoT, and it has confused both of them too (and still remains unnamed in both of them). Unfortunately, MM was unable to give any insights into this data and it's still just as confusing